PowerPointLabs/PowerPointLabs/ActionFramework/TooltipsLab/AssignTooltip/AssignTooltipEnabledHandler.cs

```diff
@@ -14,7 +14,14 @@ class AssignTooltipEnabledHandler : EnabledHandler
         protected override bool GetEnabled(string ribbonId)
         {
             Selection currentSelection = this.GetCurrentSelection();
-            return ShapeUtil.IsSelectionMultipleSameShapeType(currentSelection);
+            if (!ShapeUtil.IsSelectionShape(currentSelection))
```

Comment:
Is there a reason why `ShapeUtil.IsSelectionShape(currentSelection)` is not returned directly?